src/graph_compute/compute.cpp

```diff
@@ -12,46 +12,69 @@
 #include "./common.h"
 #include "./io.h"
 #include "./update_function.h"
+#include "./concurrent_queue.h"
 
 using namespace std;
 
-#if BASELINE || D1_PRIO
-  #include "./priority_scheduling.h"
+#if BASELINE || D0_BSP
+  #include "./bsp_scheduling.h"
 #elif D1_CHUNK
   #include "./chunk_scheduling.h"
 #elif D1_PHASE
   #include "./phase_scheduling.h"
-#elif D0_BSP
-  #include "./bsp_scheduling.h"
+#elif D1_NUMA
+  #include "./numa_scheduling.h"
+#elif D1_PRIO
+  #include "./priority_scheduling.h"
 #else
-  #error "No scheduling type defined! Specify one of BASELINE, D0_BSP, D1_PRIO, D1_CHUNK."
+  #error "No scheduling type defined!"
+  #error "Specify one of BASELINE, D0_BSP, D1_PRIO, D1_CHUNK, D1_PHASE, D1_NUMA."
 #endif
 
-WHEN_TEST(
-  static uint64_t roundUpdateCount = 0;
-)
-
 // fill in each node with random-looking data
 static void fillInNodeData(vertex_t * nodes, const vid_t cntNodes) {
-  cilk_for (vid_t i = 0; i < cntNodes; ++i) {
+  for (vid_t i = 0; i < cntNodes; ++i) {
```

Comment:
I want to remove cilk from all but the timed scheduling-specific routines, so that there are no issues with the creation of pthreads in the numa_scheduling code.